### ISSUE TITLE
export all existing chat

### ISSUE THREAD

**Issue by masbudikusuma** [chrishubert/whatsapp-api#177]
HI,
Can I export all existing chats? I tried the "/client/getChats/{sessionId}" feature only today?

**Comment by avoylenko** [comment]
The `/client/getChats/{sessionId}` endpoint is exactly what you need.

**Comment by SnakeO** [comment]
Does fetchMessages return all messages, regardless if there's 1 or 100,000 ?

**Comment by avoylenko** [comment]
@SnakeO yes, it should return all messages that's why you have to pass the `searchOptions` option to apply the limit.

See https://docs.wwebjs.dev/Chat.html#fetchMessages
